disassemblers/ofrak_pyghidra/ofrak_pyghidra/standalone/pyghidra_analysis.py

```diff
@@ -49,12 +49,20 @@ def unpack(program_file, decompiled, language=None, base_address=None):
             main_dictionary[seg_key] = code_region
             func_cbs = _unpack_code_region(code_region, flat_api)
             code_region["children"] = []
+
+            from ghidra.app.decompiler import DecompInterface
```

Comment:
I really don't like imports inside of functions without a good reason, let alone repeatedly importing this inside a loop body! 

Not sure how I missed this on the initial review of this code, but we should not repeat this mistake without a really good reason.